docs/public-networks/reference/api/index.md

```diff
@@ -1435,17 +1435,30 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_resyncWorldState","params"
 
 ### `debug_setHead`
 
-Sets the current head of the local chain to the block matching the specified block number.
+Sets the local chain head to the specified block. Optionally, moves the [bonsai](../../concepts/data-storage-formats.md#bonsai-tries)
+world state to that block.
+
+Moving the world state allows expensive operations like [`debug_traceBlock`](#debug_traceblock)
+to run on historical blocks without replaying all intermediate states. This is helpful to avoid
+out of memory errors when executing RPC calls on historical states.
+
+:::caution
+This method should not be used when a consensus client is directing Besu, or while the node is
+actively importing or proposing blocks.
```

Comment:
Maybe this can be a bit scarier also.  It is likely to cause database corruption if the node is trying to import or process blocks at all.  Not just execution engine, any change to the chain head or worldstate from any consensus mechanism will cause problems while this call is executing.

One other callout here might be that the RPC operation might time out if the distance the head is moving is more than 5000 blocks or so.  The operation will continue to run on besu, but the RPC response itself might time out